### DISCUSSION TITLE
Drawing primitives

### DISCUSSION THREAD

**Discussion by underdoeg** [tgfrerer/island#14] [Q&A]
Is there already a 2d drawing primitive to quickly draw in image (aka textured rect)?

**Comment by tgfrerer** [comment]
Not yet - but for an example of how to render a textured quad, the lut grading example code might help: https://github.com/tgfrerer/island/blob/2b718dc7bdb16fc4835c1b5bf9b8a6b9b1cb2e67/apps/examples/lut_grading_example/lut_grading_example_app/lut_grading_example_app.cpp#L158-L186
